src/mmframe.cpp

```diff
@@ -2458,10 +2458,11 @@ bool mmGUIFrame::openFile(const wxString& fileName, bool openingNew, const wxStr
             db_lockInPlace = Model_Infotable::instance().getBool("ISUSED", false);
             if (db_lockInPlace) {
                 int response = wxMessageBox(_(
-                    "The database you are trying to open has been marked as opened by another instance of MMEX.\n"
+                    "Attempting to open a database that has been marked as opened by another instance of MMEX.\n"
                     "To avoid data loss or conflict, it's strongly recommended that you close all other applications that may be using the database.\n\n"
-                    "If nothing else is running, it's possible that the database was left open as a result of a crash during previous usage of MMEX.\n\n"
-                    "Would you like to continue to open this database?")
+                    "If nothing else is running, it's possible that the database was left open as a result of a crash during previous usage of MMEX.") +
+                    "\n\n" +
+                    _("Do you want to open the database?"))
```

Comment:
There is an extra closing parenthesis here, this is causing the build failures.

Reply:
Done, thank you.